err_stash.py

```diff
@@ -734,11 +733,9 @@ def get_configuration_template(self):
             "GITHUB_ORGANIZATIONS": None,
         }
 
-    def load_user_settings(self, user, additional_settings: dict = None):
+    def load_user_settings(self, user):
         key = "user:{}".format(user)
-        settings = {"token": "", "github_token": ""}
-        if additional_settings:
-            settings.update(additional_settings)
+        settings = {"token": "", "github_token": "", "delete-branches": {}}
```

Comment:
Thought this was more secure way to do, and i was enabled to validate `delete-branches` as `dict`

Reply:
I don't get it, why `delete-branches` has to be part of the user `settings`?
We also need to make sure this `dict` isn't kept as part of the user settings after it was used, because other user settings (`token` and `github_token`) are always kept, the user only configures them once.

Reply:
What do you guys think?
keep it like it is or change to another file? doing the pick and dump manually?

Reply:
I think changing to another file is OK I guess